### DISCUSSION TITLE
Ingin menerbitkan web publik

### DISCUSSION THREAD

**Discussion by Septiad** [community/community#33974] [Other Feature Feedback, Questions, & Ideas]
Bagai mana caranya untuk menerbitkan web yg sudah saya buat HTMLnya

**Comment by DylanDevelops** [comment]
If you would like to publish html to the web and use GitHub pages to do so, you would create a repository and follow the instructions in this guide: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages.

I believe this is what you are asking! (I used google translate so let me know if this is not what you were asking for!)
